css-shapes-2/Overview.bs

```diff
@@ -406,6 +406,120 @@ The ''shape()'' Function</h4>
 			which affects how line-joins and line-caps are rendered.
 	</dl>
 
+<h5 id=shape-examples>Using ''shape()'' to create responsive, parametric speech bubble</h5>
+
+<div class=example>
+The ''shape()'' function enables shapes that are responsive, rather than scalable.
+While the ''polygon()'' shape is also responsive, it doesn't support curves.
```

Comment:
Great example!

polygon adds the `round` keyword, which I think may not be accurately described here.
https://drafts.csswg.org/css-shapes-1/#funcdef-basic-shape-polygon

Reply:
True, I'll update!